hasher-matcher-actioner/hma-lite/API.md

```diff
@@ -0,0 +1,29 @@
+Here's a super-rough first cut for an API
+
+# Match a single hash against the index
+
+```
+GET /v1/hashes/query?hash=hash_in_hex_string
+```
+
+`200 OK` if found, `404 Not Found` if not.
+
+# Match multiple hashes against the index
+
+```
+POST /v1/hashes/query
+{
+    "hashes": [
+        ...hashes in hex string
+    ]
+}
```

Comment:
Think that this is okay to drop on first pass, but I think there's low cost to including